lib/openshift/route.rb

```diff
@@ -106,5 +106,13 @@ def dns(by: nil, user: nil, cached: true, quiet: false)
       return raw_resource(user: user, cached: cached, quiet: quiet).
         dig("spec", "host")
     end
+    # check for ingress route only
+    def ready?(user: nil, quiet: false)
```

Comment:
cached can be true here like other methods. Often times you get object and call different methods. No need to treat `ready?` differently